app/workers/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -42,38 +57,41 @@ class SubmitForm0781 < Job
       # `file`, which is the generated file location
       def get_docs(submission_id, uuid)
         @submission_id = submission_id
-        parsed_forms = JSON.parse(submission.form_to_json(Form526Submission::FORM_0781))
-        file_type_and_file_objs = []
+        @uuid = uuid
+
         { 'form0781' => FORM_ID_0781, 'form0781a' => FORM_ID_0781A }.each do |form_type_key, actual_form_types|
           if parsed_forms[form_type_key].present?
             file_type_and_file_objs << {
               type: actual_form_types,
-              file: process_0781(submission.auth_headers, uuid, FORM_ID_0781, parsed_forms[form_type_key],
+              file: process_0781(uuid, FORM_ID_0781, parsed_forms[form_type_key],
                                  upload: false)
             }
           end
         end
         file_type_and_file_objs
       end
 
+      def parsed_forms
+        @parsed_forms ||= JSON.parse(submission.form_to_json(Form526Submission::FORM_0781))
+      end
+
       # Performs an asynchronous job for generating and submitting 0781 + 0781A PDF documents to VBMS
       #
       # @param submission_id [Integer] The {Form526Submission} id
       #
       def perform(submission_id)
+        @submission_id = submission_id
```

Comment:
assigning this to an attribute / instance var makes it available to the instance level logger